introduction.md

```diff
@@ -4,3 +4,10 @@ Currently, I am learning Git at CodeSchool.
 My favorite Code School path is Electives.
```

Comment:
Since you're becoming a GitHub master, could you also add "Killing history using git rebase" as one of your deadly skills?